shared/js/blobview.js

```diff
@@ -420,33 +333,36 @@ var BlobView = (function() {
     },
 
     // Read bytes up to and including a null terminator, but never
-    // more than size bytes.  And return as a Latin1 string
-    readNullTerminatedLatin1Text: function(size) {
-      var s = '';
-      for (var i = 0; i < size; i++) {
-        var charcode = this.view.getUint8(this.index + i);
-        if (charcode === 0) {
-          i++;
-          break;
-        }
-        s += String.fromCharCode(charcode);
+    // more than size bytes.  And return as an ASCII string
+    readNullTerminatedASCIIText: function(size) {
```

Comment:
I can't remember if there was any reason for ASCII vs Latin1. Looking at it now, it seems to me that "BinaryString" would be more accurate in both cases.  To avoid possible changes in semantics I'd recommend sticking with the String.fromCharCode() implementation. But if you want to rename the method to use BinaryString that's okay with me.